airbyte-webapp/src/packages/cloud/views/auth/components/Header/Header.module.scss

```diff
@@ -1,10 +1,12 @@
 @use "../../../../../../scss/colors";
+@use "../../../../../../scss/variables";
 
 .links {
   width: 100%;
   display: flex;
   flex-direction: row;
   justify-content: flex-end;
+  margin-bottom: variables.$spacing-xl;
```

Comment:
:information_source: Give the top button (switch between login/signup) some padding to the bottom so the main content doesn't squish into it too much.